src/Libraries/RevitNodes/Elements/Floor.cs

```diff
@@ -196,34 +196,39 @@ public IEnumerable<Pt> Points
         }
 
         /// <summary>
-        /// Add Point to Slab Shape
+        /// Add a new point to the slab shape editor.
+        /// Behaves the same as adding a point manually to a slab in
+        /// Revit's slab shape edit mode.
         /// </summary>
-        public void AddPoint(Pt point)
+        public static Floor AddPoint(Floor floor, Pt point)
```

Comment:
Sorry if I am not making it very clear. But you can refer FamilyInstance.SetRotation. You do not need to pass in a floor as a parameter.